### PR TITLE
install scripts for windows

### PR DESCRIPTION
**Pull request by Zonbi-san**
this script assumes you have mingw64 already installed and added to environmental variables but otherwise it should be mostly painless to install

### CODE REVIEW

File: installers/opencv.ps1 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Expand-Archive -Path ./opencv.zip
+Expand-Archive -Path ./opencv_contrib.zip
+#downloading opencv and extracting it
+
+New-Item -ItemType Directory -Path ./build
+set-location ./build
+
+$CMAKE_OPTIONS = '-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF  -DWITH_CUDA:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF -DINSTALL_CREATE_DISTRIB=ON'
+$CMAKE_GENERATOR_OPTIONS = '-G "Ninja"' #switch to whatever generator you use but the ones with long names is not working for some stupid reason
+$P_REPO_LOCATION = 'C:\Users\ourpc\CLionProjects\Polarity-' #change ya path to whereever polarity root folder is
```
Comment (Ashthetik) [inline installers/opencv.ps1]: You could just change this to `Get-Location` tbh
Saves the user needing to edit the file and potentially getting the directory wrong

File: installers/opencv.ps1 [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+wget -O opencv.zip https://github.com/opencv/opencv/archive/4.x.zip
+Expand-Archive -Path ./opencv.zip
+Expand-Archive -Path ./opencv_contrib.zip
+#downloading opencv and extracting it
+
+New-Item -ItemType Directory -Path ./build
+set-location ./build
+
+$CMAKE_OPTIONS = '-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF  -DWITH_CUDA:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF -DINSTALL_CREATE_DISTRIB=ON'
+$CMAKE_GENERATOR_OPTIONS = '-G "Ninja"' #switch to whatever generator you use but the ones with long names is not working for some stupid reason
```
Comment (Ashthetik) [inline installers/opencv.ps1]: For Windows, I'd say staying with their default compiler (`cl` or `clang-cl`)

**Review by Ashthetik — CHANGES_REQUESTED** [review]

**Comment by Zonbi-san** [comment]
hmm alright gonna see what i can do

**Comment by Zonbi-san** [comment]
alright finished all of your requested changes

**Comment by Ashthetik** [comment]
lgtm, I'll double check in the morning and merge it.

as comfy said: it's bad luck to merge at night lmao

File: installers/armadillo.ps1 [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#winget install Microsoft.VisualStudio.2022.BuildTools Kitware.CMake Git.Git
+
+#git clone https://gitlab.com/conradsnicta/armadillo-code.git
```
Comment (Ashthetik) [inline installers/armadillo.ps1]: I think you forgot to uncomment this lmao

**Review by Ashthetik — CHANGES_REQUESTED** [review]
lgtm so far just needs the git clone uncommented so the armadillo installer doesn't error out

File: installers/armadillo.ps1 (hunk 3 above)
Comment (Zonbi-san) [reply installers/armadillo.ps1]: (ˉ▽ˉ；)...
yeah... just fixed it

**Review by Ashthetik — APPROVED** [review]

**Comment by Ashthetik** [comment]
merging now 👍

**Pull request merged**